distribution/docs/src/main/resources/content/_reference/_tables/OpenSearchSource.adoc

```diff
@@ -65,6 +65,20 @@
 |true
 |true
 
+|Multi Point-Radius polygon approximation vertices
+|[[_numMultiPointRadiusVertices]]numMultiPointRadiusVertices
+|Integer
+|When performing a multi point-radius search, increase this value for more accurate polygon approximation. Minimum value is 4, maximum value is 32.
+|true
```

Comment:
This column is `Default Value`, so these rows should have `32` and `1`.

Reply:
whoops, good catch